src/wasm-builder.h

```diff
@@ -629,6 +629,19 @@ class Builder {
     ret->finalize();
     return ret;
   }
+  I31New* makeI31New(Expression* value) {
+    auto* ret = allocator.alloc<I31New>();
```

Comment:
`Builder` no longer has an `allocator`, so you will need to merge master and update this.